liblangutil/ErrorReporter.h

```diff
@@ -96,6 +98,17 @@ class ErrorReporter
 
 	void typeError(ErrorId _error, SourceLocation const& _location, std::string const& _description);
 
+	template <typename... Args>
+	void typeError(ErrorId _error, SourceLocation const& _location, std::string const& _descriptionFmt, Args... _args)
+	{
+		error(
+			_error,
+			Error::Type::TypeError,
+			_location,
+			fmt::format(_descriptionFmt, std::forward<Args>(_args)...)
+		);
+	}
```

Comment:
Not sure if this is a good thing. When I looked at the calls, the extra parameters to `typeError()` looked a bit confusing. I think I'd prefer to see an explicit call to `fmt::format`.

Reply:
I'm absolutely fine in dropping that and using fmt::format() more explicit instead. - I was adding that in the hope of improving the use of it. Whatever we come up with, I am fine.